make/harbor.yml.tmpl

```diff
@@ -59,6 +59,22 @@ clair:
   # The interval of clair updaters, the unit is hour, set to 0 to disable the updaters.
   updaters_interval: 12
 
+# Trivy configuration
+trivy:
+  # github_token The GitHub access token to download Trivy DB
+  #
+  # Trivy DB contains vulnerability information from NVD, Red Hat, and many other upstream vulnerability databases.
+  # It is downloaded by Trivy from the GitHub release page https://github.com/aquasecurity/trivy-db/releases and cached
+  # in the local file system (/home/scanner/.cache/trivy/db/trivy.db). In addition, the database contains the update
+  # timestamp so Trivy can detect whether it should download a newer version from the Internet or use the cached one.
+  # Currently, the database is updated every 12 hours and published as a new release to GitHub.
+  #
+  # Anonymous downloads from GitHub are subject to the limit of 60 requests per hour. Normally such rate limit is enough
+  # for production operations. If, for any reason, it's not enough, you could increase the rate limit to 5000
+  # requests per hour by specifying the GitHub access token. For more details on GitHub rate limiting please consult
```

Comment:
How can the users get the access token and from where to get it (Trivy-DB repo)?

Reply:
> How can the users get the access token and from where to get it (Trivy-DB repo)?

You can generate the GitHub token as described here: https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line

The token is used solely for rate limiting not for managing access rights. There's nothing specific to GitHub and Trivy. We just happen to store the trivy-db.tar.gz artifact on GitHub and download it through GitHub API

Reply:
@steven-zou I've added a link to GitHub page explaining how to create an access token.